src/System.Collections.Specialized/src/System/Collections/Specialized/NameObjectCollectionBase.cs

```diff
@@ -90,56 +90,12 @@ protected NameObjectCollectionBase(int capacity)
 
         protected NameObjectCollectionBase(SerializationInfo info, StreamingContext context)
         {
-            _serializationInfo = info;
+            throw new PlatformNotSupportedException();
```

Comment:
I don't have (and can't quickly get) base type data. However, I did a quick GitHub search and found one instance of a serializable derived type. Additionally, since this is more or less a collection, it fits our general model, so I've added it back.